### PR TITLE
[BUG] Modified the background image to be hi-res and added image credits

### LINKED ISSUE #311: [BUG] Background Image is Low-Res
#### Describe the bug:
The current background image is very low res and doesn't look good. We should replace it with a much higher-res version 

### To Reproduce the bug:
A clear description of the steps to reproduce the behaviour:

1. Just open the app
2. Look at all the pixels

### Additional context:
High Res version: https://whitecoconutthai.com/wp-content/uploads/2020/02/AdobeStock_297677001-scaled.jpeg

### PR DESCRIPTION
**Pull request by SreenikethRaghavan**
Closes #311

### CODE REVIEW

**Review by reeved — APPROVED** [review]
Changes look good. Nice work, Sree :)

**Review by UltimaKuma — APPROVED** [review]
LGTM

File: frontend/src/App.css [hunk 1]
```diff
@@ -50,7 +50,7 @@
 
 .BackGroundImage {
   background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
-    url('./backgroundImg.jpeg');
+    url('./backgroundImg.jpeg'); /*Image credits: https://unsplash.com/*/
```
Comment (Dhors) [inline frontend/src/App.css:53]: Nice find Sree. The artist wasn't credited before which would have been unfair to them.

**Review by Dhors — APPROVED** [review]
Nice find Sree. 

The artist wasn't credited before which would have been unfair to them.

**Pull request merged**